Kinvey/Kinvey/FileStore.swift

```diff
@@ -219,7 +219,13 @@ open class FileStore {
                 }
                 
                 let handle: (Data?, URLResponse?, Swift.Error?) -> Void = { data, response, error in
-                    if let response = response as? HTTPURLResponse , 200 <= response.statusCode && response.statusCode < 300 {
+                    if self.client.logNetworkEnabled, let response = response as? HTTPURLResponse {
```

Comment:
@heyzooi Shouldn't this `logNetworkEnabled` check be a level lower i.e. at the request level?